lib/get-group-tangle.js

```diff
@@ -42,18 +41,24 @@ module.exports = function GetGroupTangle (server, keystore) {
     }
   }
 
-  return function getGroupTangle (groupId, cb) {
+  return function getGroupTangle (groupId, tangle, cb) {
+    if (typeof tangle === 'function') {
+      cb = tangle
+      tangle = 'group'
+    }
     if (!isGroup(groupId)) return cb(new Error(`get-group-tangle expects valid groupId, got: ${groupId}`))
 
     const info = keystore.group.get(groupId)
     if (!info) return cb(new Error(`get-group-tangle: unknown groupId ${groupId}`))
 
     // if it's in the cache, then get the cached value, then callback
     if (cache.has(groupId)) {
```

Comment:
:fire: the cache on line `:11` is designed to track **group** tangle updates.

It won't track membership updates, but the code you've written WILL think it has ? and give `previous` (tips) info based on that D:

So .... you are gonna need 2 caches `groupTangleCache` and `memberTangleCache`

Reply:
TODO move the tangle arg up to the GetGroupTangle constructor

Reply:
keeping 1 cache per GetGroupTangle, but instead using 1 GetGroupTangle per tangle (so 2 in the publish hook)